bot/bot.py

```diff
@@ -97,4 +105,18 @@ async def wait_until_guild_available(self) -> None:
         await self._guild_available.wait()
 
 
-bot = SirRobin(command_prefix=constants.Client.prefix, activity=disnake.Game("The Not-Quite-So-Bot-as-Sir-Lancebot"))
+_intents = discord.Intents.default()  # Default is all intents except for privileged ones (Members, Presences, ...)
```

Comment:
Seeing how many we're disabling. Would it be easier using `discord.Intents.none()`?

Reply:
I mean that's how it's done on Lance, it's a bit more easier to understand this way, the result is basically the same. :D